specification/chaos/resource-manager/Microsoft.Chaos/preview/2022-07-01-preview/experiments.json

```diff
@@ -0,0 +1,526 @@
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Experiment resource returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experiment"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a Experiment in a resource group.": {
+            "$ref": "./examples/GetAExperiment.json"
+          }
+        }
+      },
+      "put": {
+        "description": "Create or update a Experiment resource.",
+        "tags": [
+          "Experiments"
+        ],
+        "operationId": "Experiments_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "./types/common.json#/parameters/apiVersion"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "./types/common.json#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "./types/experiments.json#/parameters/experimentName"
+          },
+          {
+            "name": "experiment",
+            "description": "Experiment resource to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experiment"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Experiment resource returned if request was successful.",
+            "schema": {
+              "$ref": "./types/experiments.json#/definitions/experiment"
+            }
+          },
+          "default": {
+            "description": "Error response returned if request was unsuccessful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create/update a Experiment in a resource group.": {
+            "$ref": "./examples/CreateOrUpdateAExperiment.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Chaos/experiments/{experimentName}/cancel": {
+      "post": {
+        "description": "Cancel a running Experiment resource.",
```

Comment:
this operatino used to have 'x-ms-longrunning-operation': true, why remove it ?

Reply:
@jianyexi, on the server side, we don't implement this operation as a LRO, and are removing the header so that the Swagger aligns with the implementation. This was a suggestion made to us by @raych1 in a thread offline.

Reply:
but this change is breaking change for SDK , so you need to follow the breaking change review process